store/src/migrations.rs

```diff
@@ -42,8 +42,6 @@ pub static MIGRATIONS: Lazy<Migrations> = Lazy::new(|| {
             block_num INTEGER NOT NULL,
 
             PRIMARY KEY (account_id),
-            CONSTRAINT accounts_block_account_id_is_felt CHECK (account_id >= 0 AND account_id < 18446744069414584321),
-            CONSTRAINT accounts_block_num_is_u32 CHECK (block_num >= 0 AND block_num < 4294967296),
```

Comment:
It seems like enabling `account_id` constraints may be problematic because sqlite doesn't have native `u64` support. But do we also need to disable `block_num` constraints?

Reply:
oops, added it back